lessons/models.py

```diff
@@ -80,33 +86,42 @@ class Lesson_request(models.Model):
     availableTimes = models.CharField(max_length=255, blank=False, choices=TIME_CHOICES)
     numberOfLessons = models.PositiveIntegerField(blank=False, validators=[validate_nonzero])
     IntervalBetweenLessons = models.PositiveIntegerField(blank=False, validators=[validate_nonzero])
-    DurationOfLesson = models.PositiveIntegerField(blank = False, validators=[validate_nonzero], choices=VALUE_CHOICES)
+    DurationOfLesson = models.PositiveIntegerField(blank=False, validators=[validate_nonzero], choices=VALUE_CHOICES)
     LearningObjectives = models.TextField(blank=False)
     AdditionalNotes = models.TextField(blank=True)
     Fulfilled = models.CharField(max_length=50, blank=False, default='Pending')
 
     def getWeekdays_choices():
         return WEEKDAY_CHOICES
-    
+
     def getTime_choices():
         return TIME_CHOICES
 
+
 class Invoice(models.Model):
     """Invoice model used to create invoices of bank transfers for the lesson payments"""
 
     reference = models.CharField(
-        unique=True, 
-        blank=False, 
-        max_length=50, 
-        primary_key=True, 
-        validators= [RegexValidator(
+        unique=True,
+        blank=False,
+        max_length=50,
+        primary_key=True,
+        validators=[RegexValidator(
             regex='^[0-9-]*$',
-            message= 'Reference number should be numbers-numbers',
+            message='Reference number should be numbers-numbers',
             code='invalid_reference'
         )]
     )
-    price = models.FloatField(blank=False, validators= [MaxValueValidator(1000000)])
-    unpaid = models.FloatField(blank=False, validators= [MaxValueValidator(1000000)])
+    price = models.FloatField(blank=False, validators=[MaxValueValidator(1000000)])
+    unpaid = models.FloatField(blank=False, validators=[MaxValueValidator(1000000)])
     creation_date = models.DateTimeField(blank=False, validators=[MaxValueValidator(limit_value=timezone.now)])
     update_date = models.DateTimeField(blank=False, validators=[MaxValueValidator(limit_value=timezone.now)])
     student = models.ForeignKey(User, on_delete=models.CASCADE, blank=False)
+
+
+class Term(models.Model):
+    """Term model used to keep track of lessons per term"""
+    termID = models.IntegerField(unique=True, blank=False)
+    startDate = models.DateField(default=datetime.date.today())
```

Comment:
The default should be `timezone.now` cause what is added now raises a warning. 

Also, for the dates should have a validator that doesn't allow dates in the past